src/containers/board/bestArticleList/bestArticleList.tsx

```diff
@@ -1,48 +1,28 @@
-import { useState, useEffect } from 'react';
 import useBestArticles from '@/hooks/useBestArticles';
 import BestArticleCard from './bestArticleCard/bestArticleCard';
 
 function BestArticleList() {
-  const [pageSize, setPageSize] = useState<number>(1);
-
-  const handleResize = () => {
-    const width = window.innerWidth;
-    if (width < 768) {
-      setPageSize(1);
-    } else if (width < 1200) {
-      setPageSize(2);
-    } else {
-      setPageSize(3);
-    }
-  };
-
-  useEffect(() => {
-    handleResize();
-    window.addEventListener('resize', handleResize);
-    return () => window.removeEventListener('resize', handleResize);
-  }, []);
-
-  // 커스텀 훅을 사용하여 데이터를 가져옴
-  const { data, isLoading, isError } = useBestArticles(pageSize);
+  const { data, isLoading, isError } = useBestArticles(3); // 최대 3개의 데이터를 요청
```

Comment:
여기서만 사용되는 별거 아닌 숫자지만 매직 넘버는 사용 지양하고 상수로 사용해주세요~!

[1. 의미가 불분명한 매직 넘버를 상수로 선언하라.](https://javabom.tistory.com/28)

```suggestion
  const BEST_ARTICLE_SIZE = 3;
  const { data, isLoading, isError } = useBestArticles(BEST_ARTICLE_SIZE); // 최대 3개의 데이터를 요청
```